codalab/apps/api/tests.py

```diff
@@ -158,10 +167,18 @@ def test_attempting_to_join_competition_auto_approved_sends_emails(self):
 
         subjects = [m.subject for m in mail.outbox]
         self.assertIn('Accepted into Test Competition!', subjects)
-        self.assertIn('%s accepted into your competition!' % self.participant_user, subjects)
+        self.assertIn(
+            '%s accepted into your competition!' %
+            self.participant_user,
+            subjects)
 
-    def test_attempting_to_join_competition_not_logged_in_doesnt_send_email(self):
-        resp = self.client.post(reverse('competition-participate', kwargs={'pk': self.competition.pk}))
+    def test_attempting_to_join_competition_not_logged_in_doesnt_send_email(
```

Comment:
I noticed a lot of changes like these that spread out the lines, which doesn't actually seem to make it easier to read.  I guess this is because we have a 80 column width limit or something...